### PR TITLE
No blackhole you can't eat that ghost

### PR DESCRIPTION
**Pull request by flappybatpal**
[BUG]

## About the PR 
Stops blackholes pulling ghost things.
Makes blackhole singularity immune.

## Why's this needed? 
Leave my ghost alone!
Removes a check. Does stop black holes pulling each other together but A) How often is that going to happen B) I tested it and it's not very cool

### CODE REVIEW

File: code/modules/events/black_hole.dm [hunk 1]
```diff
@@ -142,7 +143,7 @@
 			return
 
 		for (var/atom/X in range(7,src))
-			if (X == src || (X.event_handler_flags & IMMUNE_SINGULARITY))
+			if (X.event_handler_flags & IMMUNE_SINGULARITY)
```
Comment (mordent-goonstation) [inline code/modules/events/black_hole.dm:146]: The two changes in this file mean these black hole things will not pull each other towards each other, I believe? Verifying if intended behavior, as not part of the PR description. Doesn't seem like a big deal, just noting.

Comment (flappybatpal) [reply code/modules/events/black_hole.dm:146]: It's part of the PR description.

**Review by ZeWaka — APPROVED** [review]

**Review by pali6 — APPROVED** [review]
What about intangible mobs?

**Comment by flappybatpal** [comment]
> What about intangible mobs?

The base type /mob/living/intangible has event_handler_flags =  IMMUNE_MANTA_PUSH | IMMUNE_SINGULARITY
